android/src/main/java/com/idehub/GoogleAnalyticsBridge/GoogleAnalyticsBridge.java

```diff
@@ -481,4 +482,11 @@ public void createNewSession(String trackerId, String screenName) {
         tracker.setScreenName(screenName);
         tracker.send(new HitBuilders.ScreenViewBuilder().setNewSession().build());
     }
+
+    @ReactMethod
+    public void dispatch(Promise promise) {
+        GoogleAnalytics analytics = GoogleAnalytics.getInstance(getReactApplicationContext());
```

Comment:
There's already a convenience method for getting the instance, please result that: https://github.com/idehub/react-native-google-analytics-bridge/blob/cceff5435dc5dda28efb423c788096262977cd9c/android/src/main/java/com/idehub/GoogleAnalyticsBridge/GoogleAnalyticsBridge.java#L46

E.g. `GoogleAnalytics analytics = getAnalyticsInstance();`